<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>780</width>
      <height>550</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_dashboard">
     <attribute name="title">
      <string>Dashboard</string>
     </attribute>
     <widget class="QLabel" name="label_dashboard">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>200</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Gauges Here</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_ve_table">
     <attribute name="title">
      <string>VE Table</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_ve">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>750</width>
        <height>500</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_ignition_table">
     <attribute name="title">
      <string>Ignition Table</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_ignition">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>750</width>
        <height>500</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_logs">
     <attribute name="title">
      <string>Logs</string>
     </attribute>
     <widget class="QTextEdit" name="textEdit_logs">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>750</width>
        <height>500</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
